compiler/control/OMROptions.cpp

```diff
@@ -1949,6 +1951,15 @@ OMR::Options::Options(
    if (_suppressLogFileBecauseDebugObjectNotCreated && !optimizationPlan->isLogCompilation())
       _logFile = NULL;
 
+   if (self()->getOption(TR_EnableCodeCacheDisclaimingSupport))
+      {
```

Comment:
Typically we setup options internally in `jitLatePostProcess`.

Reply:
Fixed